lexxpluss_apps/src/imu_controller.cpp

```diff
@@ -131,6 +131,15 @@ class imu_fetcher {
                     gyro_data[0] = gyro_rad_to_deg_int16_t(&gyro[1]);
                     gyro_data[2] = gyro_rad_to_deg_int16_t(&gyro[2]);
 
+                    if(imu_bias_initial_cnt < IMU_BIAS_INITIAL_CNT_MAX) {
+                        bias_gyro_z = gyro_data[2];
```

Comment:
Is this the intended behavior of using gyro_data taken at the end as bias?
I was wondering if it would be better to take an average or something.
Since the title says “draft”, I think it means that bias will be improved in the future.

一番最後にとったgyro_dataをbiasとして使用するで意図通りでしょうか?
平均を取ったりした方が良かったりしないかというのが気になった点です。
タイトルにdraftとあるので、今後biasについては改良していくということだとは思います。